src/components/Button.js

```diff
@@ -27,28 +27,19 @@ export default function Button({
   let triggeredLabel = '';
   switch (type) {
     case 'save':
-      label = labels.save.label;
-      triggeredLabel = labels.save.triggeredLabel;
-      break;
     case 'create':
-      label = labels.create.label;
-      triggeredLabel = labels.create.triggeredLabel;
-      break;
-    case 'delete':
-      label = labels.delete.label;
-      break;
-    case 'view':
-      label = labels.view.label;
-      break;
-    case 'upload':
-      label = labels.upload.label;
+      label = labels[type]['label'];
```

Comment:
No need to use brackets for literals.
```suggestion
      label = labels[type].label;
```

Reply:
Yes, that is evident from the deleted lines.
I chose to use bracket-notation with string literals to avoid mixing the two syntaxes together.

I'll make the changes..